x-pack/plugins/case/server/client/cases/update.test.ts

```diff
@@ -38,7 +38,10 @@ describe('update', () => {
       });
 
       const caseClient = await createCaseClientWithMockSavedObjectsClient(savedObjectsClient);
-      const res = await caseClient.client.update({ cases: patchCases });
+      const res = await caseClient.client.update({
+        caseClient: caseClient.client,
+        cases: patchCases,
```

Comment:
It appears you need to pass a mock here for `context?.securitySolution?.getAppClient()`, otherwise it blows up jest:
```
(node:28367) UnhandledPromiseRejectionWarning: TypeError: Cannot read property 'securitySolution' of undefined
```

Reply:
I take it back, this is not where you need to mock it... but you do need to mock it!